back/config/initializers/rack_attack.rb

```diff
@@ -110,14 +110,26 @@ class Rack::Attack
   # Resend code by IP.
   throttle('user/resend_code', limit: 10, period: 5.minutes) do |req|
```

Comment:
I think `resend_code/ip` instead of `user/resend_code` would be more consistent with the other entries.

Reply:
(sorry I clicked "add single comment" instead of "start a review" but still reviewing)